test/groovy/org/ods/component/ContextSpec.groovy

```diff
@@ -185,7 +185,7 @@ class ContextSpec extends Specification {
     }
 
     Map<String, String> getCloneProjectScriptUrls(config) {
-        def uut = new Context(script, config, logger)
+        def IContext uut = new Context(script, config, logger)
```

Comment:
`def IContext` should be just `IContext` I guess?

Reply:
nope .. it looks like it's reusing the same context from above .. and does not find the method. I have no idea - why this ever worked, for now it doesnt :(